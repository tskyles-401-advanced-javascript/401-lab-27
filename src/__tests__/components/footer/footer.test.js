import React from 'react';
import Footer from '../../../components/footer/footer';

describe("<Footer />", () => {
  it('is alive at app start', () => {
    let app = shallow(<Footer/>);
    expect(app.find('footer').exists()).toBeTruthy();
  });
  it('renders correctly', () => {
    const tree = renderer.create(<Footer/>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});