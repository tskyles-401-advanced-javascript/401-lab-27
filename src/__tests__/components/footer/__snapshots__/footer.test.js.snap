// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> renders correctly 1`] = `
<footer>
  Â© 2018 Code Fellows
</footer>
`;
